src/Examples/Flowsheets/temperature_swing_adsorption/TSA_CO2_example_1DFixedBed.ipynb

```diff
@@ -0,0 +1,1961 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "66da77e5",
+   "metadata": {},
+   "source": [
+    "# TSA cycle of CO2 simulation example with a 1D Fixed Bed model\n",
+    "<img src='TSA_figure.png' width=\"300\" height=\"300\">\n",
+    "\n",
+    "This jupyter notebook shows an example of a temperature swing adsorption and desorption of CO2 with the IDAES 1D FixedBed model. The IDAES 1D FixedBed model is a dynamic and axially varying reactor/adsorption model which is able to model the gas and solid interactions of the modeled species in detail. The sorbent used for this example is the NETL_32D sorbent with its details and parameters obtained from the following reference: \n",
+    "- A. Lee, D.C. Miller, A One-Dimensional (1-D) Three-Region Model for a Bubbling Fluidized-Bed Adsorber, Ind. Eng. Chem. Res. 52 (2013) 469–484\n",
```

Comment:
The original paper presenting the development of the kinetic and equilibrium model for the adsorption of CO2 onto the NETL 32D sorbent should also be cited here: 

Lee, A.; Mebane, D.; Fauth, D. J.; Miller, D. C. A Model for the Adsorption Kinetics of CO2 on Amine-Impregnated Mesoporous Sorbents in the Presence of Water. Presented at the 28th International Pittsburgh Coal Conference, Pittsburgh, PA, 2011.